client/common/directives/ModelFieldImage/ModelFieldImage.js

```diff
@@ -62,6 +62,13 @@ angular.module('dashboard.directives.ModelFieldImage', [
 
         scope.uploadStatus = "Upload File";
 
+        // For question type image-display
+        if (scope.options.displayOnly) {
+          scope.displayOnly = true;
+          scope.imageUrl = scope.options.imageUrl;
+          scope.thumbnailUrl = scope.options.imageUrl;
```

Comment:
I don't remember for sure, but I think if thumbnailUrl is not provided it just defaults to loading imageUrl, so no harm in assigning it to an option that might be passed in. (Please double check)

Reply:
It doesn't. I was going to change the template to use a ternary operation on ng-style for the background image url, but it was turning out to be pretty messy. Since it's already set to load the image using `thumbnailUrl` I figure it'll just be cleanest to use that.

We shouldn't need to set `scope.imageUrl` though, so I'll remove that.

Reply:
Works for me.